Buildings/.copiedFiles.txt

```diff
@@ -2185,7 +2183,8 @@ Buildings/Resources/Images/Utilities/Math/Functions/Examples/cubicHermite.png
 Buildings/Resources/Images/Utilities/Math/Functions/SmoothExponentialXPower.png
 Buildings/Resources/Images/Utilities/Math/int.pdf
 Buildings/Resources/Images/Utilities/Math/int.png
-Buildings/Resources/Scripts/Dymola/Airflow/Multizone/BaseClasses/Examples/Interpolate.mos
+Buildings/Resources/ReferenceResults/Dymola/Buildings_Utilities_Math_Examples_Interpolate.txt
+Buildings/Resources/ReferenceResults/Dymola/Buildings_Utilities_Math_Functions_Examples_Interpolate.txt
```

Comment:
We don't include reference results in the merge (as sometimes we may differ in the Dymola version)

Reply:
Fixed in 10f14cb.
The result files in `.copiedFiles.txt` were automatically generated by the merge script.